App crash - Error inflating class android.webkit.WebView

### Actual behaviour
- Tell us what happens

### Expected behaviour
- Tell us what should happen
 
### Steps to reproduce
1. 
2. 
3. 


### Environment data
Android version:

Device model: Mate 31 PRO

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

### Logs
#### Web server error log
```
************ CAUSE OF ERROR ************

android.view.InflateException: Binary XML file line #7: Error inflating class android.webkit.WebView
	at android.view.LayoutInflater.createView(LayoutInflater.java:633)
	at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:682)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:806)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:504)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:414)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:365)
	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:555)
	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:161)
	at com.owncloud.android.authentication.AuthenticatorActivity.onGetServerInfoFinish(AuthenticatorActivity.java:1310)
	at com.owncloud.android.authentication.AuthenticatorActivity.onRemoteOperationFinish(AuthenticatorActivity.java:1209)
	at com.owncloud.android.services.OperationsService.lambda$dispatchResultToOperationListeners$0(OperationsService.java:727)
	at com.owncloud.android.services.-$$Lambda$OperationsService$2BVgsxDpOpa4JqgzeD7OGZBLSiI.run(lambda)
	at android.os.Handler.handleCallback(Handler.java:815)
	at android.os.Handler.dispatchMessage(Handler.java:104)
	at android.os.Looper.loop(Looper.java:194)
	at android.app.ActivityThread.main(ActivityThread.java:5631)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
	at android.view.LayoutInflater.createView(LayoutInflater.java:607)
	... 21 more
Caused by: android.content.res.Resources$NotFoundException: String resource ID #0x2040003
	at android.content.res.Resources.getText(Resources.java:322)
	at android.content.res.Resources.getString(Resources.java:420)
	at com.android.org.chromium.content.browser.ContentViewCore.setContainerView(ContentViewCore.java:684)
	at com.android.org.chromium.content.browser.ContentViewCore.initialize(ContentViewCore.java:608)
	at com.android.org.chromium.android_webview.AwContents.createAndInitializeContentViewCore(AwContents.java:674)
	at com.android.org.chromium.android_webview.AwContents.setNewAwContents(AwContents.java:823)
	at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:662)
	at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:597)
	at com.android.webview.chromium.WebViewChromium.initForReal(WebViewChromium.java:319)
	at com.android.webview.chromium.WebViewChromium.access$100(WebViewChromium.java:104)
	at com.android.webview.chromium.WebViewChromium$1.run(WebViewChromium.java:271)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.drainQueue(WebViewChromium.java:131)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue$1.run(WebViewChromium.java:118)
	at com.android.org.chromium.base.ThreadUtils.runOnUiThread(ThreadUtils.java:144)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.addTask(WebViewChromium.java:115)
	at com.android.webview.chromium.WebViewChromium.init(WebViewChromium.java:268)
	at android.webkit.WebView.<init>(WebView.java:591)
	at android.webkit.WebView.<init>(WebView.java:526)
	at android.webkit.WebView.<init>(WebView.java:509)
	at android.webkit.WebView.<init>(WebView.java:496)
	... 24 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30090190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Android
Device: Mate 31Pro
Model: Mate 31Pro
Id: LMY47I
Product: Mate 31Pro

************ FIRMWARE ************
SDK: 22
Release: 10.0
Incremental: 1572503998

```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

Same error when app trying to connect to a remote server. Android 5.1.
`************ CAUSE OF ERROR ************

android.view.InflateException:/nary XML file line #7: Error inflating class android.webkit.WebView
	at android.view.LayoutInflater.createView(LayoutInflater.java:633)
	at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:682)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:806)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:504)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:414)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:365)
	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:555)
	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:161)
	at com.owncloud.android.authentication.AuthenticatorActivity.onGetServerInfoFinish(AuthenticatorActivity.java:1310)
	at com.owncloud.android.authentication.AuthenticatorActivity.onRemoteOperationFinish(AuthenticatorActivity.java:1209)
	at com.owncloud.android.services.OperationsService.lambda$dispatchResultToOperationListeners$0(OperationsService.java:727)
	at com.owncloud.android.services.-$$Lambda$OperationsService$2BVgsxDpOpa4JqgzeD7OGZBLSiI.run(lambda)
	at android.os.Handler.handleCallback(Handler.java:815)
	at android.os.Handler.dispatchMessage(Handler.java:104)
	at android.os.Looper.loop(Looper.java:194)
	at android.app.ActivityThread.main(ActivityThread.java:5637)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
	at android.view.LayoutInflater.createView(LayoutInflater.java:607)
	... 21 more
Caused by: android.content.res.Resources$NotFoundException: Unable to find resource ID #0x2040003
	at android.content.res.Resources.getId(Resources.java:241)
	at android.content.res.Resources.getText(Resources.java:699)
	at android.content.res.Resources.getString(Resources.java:846)
	at com.android.org.chromium.content.browser.ContentViewCore.setContainerView(ContentViewCore.java:684)
	at com.android.org.chromium.content.browser.ContentViewCore.initialize(ContentViewCore.java:608)
	at com.android.org.chromium.android_webview.AwContents.createAndInitializeContentViewCore(AwContents.java:674)
	at com.android.org.chromium.android_webview.AwContents.setNewAwContents(AwContents.java:823)
	at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:662)
	at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:597)
	at com.android.webview.chromium.WebViewChromium.initForReal(WebViewChromium.java:319)
	at com.android.webview.chromium.WebViewChromium.access$100(WebViewChromium.java:104)
	at com.android.webview.chromium.WebViewChromium$1.run(WebViewChromium.java:271)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.drainQueue(WebViewChromium.java:131)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue$1.run(WebViewChromium.java:118)
	at com.android.org.chromium.base.ThreadUtils.runOnUiThread(ThreadUtils.java:144)
	at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.addTask(WebViewChromium.java:115)
	at com.android.webview.chromium.WebViewChromium.init(WebViewChromium.java:268)
	at android.webkit.WebView.<init>(WebView.java:590)
	at android.webkit.WebView.<init>(WebView.java:525)
	at android.webkit.WebView.<init>(WebView.java:508)
	at android.webkit.WebView.<init>(WebView.java:495)
	... 24 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30090290
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: elephone
Device: k05ts_a
Model: Elephone P8000
Id: LMY47D
Product: Elephone P8000

************ FIRMWARE ************
SDK: 22
Release: 5.1
Incremental: 1455683650
`

--- a/build.gradle
+++ b/build.gradle
@@ -290,7 +290,7 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'com.google.android.material:material:1.1.0-rc02'
     implementation 'com.jakewharton:disklrucache:2.0.2'
-    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.appcompat:appcompat:1.2.0-alpha02'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.exifinterface:exifinterface:1.1.0'
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0"

